reminder/util.py

```diff
@@ -43,7 +44,10 @@ def __init__(self, name: str, label: str = None, *, required: bool = False):
 
     def match(self, val: str, evt: MessageEvent = None, instance: 'ReminderBot' = None
               ) -> Tuple[str, Optional[datetime]]:
-        tz = pytz.UTC
+        try:
+            tz = pytz.timezone(instance.config["timezone"])
```

Comment:
This should be parsed in `ReminderBot.on_external_config_update`, e.g. `self.default_timezone = pytz.timezone(self.config["timezone"])`